### DISCUSSION TITLE
Jumper T15:  6 position switch

### DISCUSSION THREAD

**Discussion by wrevenaugh** [EdgeTX/edgetx#5482] [Q&A]
On my new JumperT15, I can't seem to figure out how to use the 6 position switch the same way I used it with my Radiomaster TX16s.  To clarify, I used to be able to select the "6P" switch as a source, then I could assign the individual buttons to flight modes in Ardupilot/INAV.  With the new T15, the "6P" is now gone and I can only find the individual buttons (now called SW1, SW2---SW6).  We can now do even more with those buttons/switches and combine them into a "Group1, 2 or 3" but then I'd expect that maybe I'd find a "source" called "GroupX" but I don't see that either.  I'm running the firmware that came with the radio "2.10.1-selfbuild".  Any help would really be appreciated.   Thanks, Bill

**Comment by pfeerick** [comment]
What you are after will be available in 2.11. i.e. this https://github.com/EdgeTX/edgetx/issues/4989

It is possible to do what you want now, but it either requires several mixer lines, or several logical switches, etc. Like shown here https://github.com/EdgeTX/edgetx/issues/4189#issuecomment-1757186041
